packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -570,6 +631,28 @@ export class EpochContext {
 
     this.effectiveBalanceIncrements[index] = Math.floor(effectiveBalance / EFFECTIVE_BALANCE_INCREMENT);
   }
+
+  /**
+   * Note: The range of slots a validator has to perform duties is off by one.
+   * The previous slot wording means that if your validator is in a sync committee for a period that runs from slot
+   * 100 to 200,then you would actually produce signatures in slot 99 - 199.
+   */
+  getIndexedSyncCommittee(slot: Slot): SyncCommitteeCache {
```

Comment:
Some consumers have to convert from epoch to slot, and this converts from slot to syncPeriod.
Should we separate this function to `getIndexedSyncCommitteeAtSlot()` and `getIndexedSyncCommitteeAtEpoch()`

Reply:
Added getIndexedSyncCommitteeAtEpoch